changelog.txt

```diff
@@ -1,3 +1,95 @@
+1.10.0-beta1 (2019-09-??)
+------------------------------------------------------------------------
+Feature: Ability to show Newspaper and Ticker messages in parallel (#7612)
+Feature: Show coverage area for stations and towns (#7446)
+Feature: Collapsible vehicle groups (#7417)
+Feature: More flexible docks - can now have multiple per station, ships can use any part of dock (#7380)
+Feature: Improved performance for road vehicle pathfinding (#7261)
+Feature: Option to show local authority boundary of towns (#7025)
+Feature: Experimental method of town cargo generation that scales linearly with population (#6965)
+Feature: [NewGRF] Houses and industries can accept/produce up to 16 different cargo types (#6867, #6872)
+Feature: [NewGRF] RoadTypes (NRT) (#6811)
+Add: [Win32] Select MIDI device by port name (#7666)
+Add: 'getsysdate' console command (#7658)
+Add: Filter on town list window (#7621)
+Add: Currencies NTD, CNY, HKD (#7596)
+Add: Icons to vehicle construction drop down lists (#7358, #7485)
+Add: [NewGRF] Railtype flags to allow/disallow 90 degree curves (#7352)
+Add: Security warning to players that company passwords are not truly secure (#7351)
+Add: Game setting to define whether industries with neutral stations (e.g. Oil rigs) accept and supply cargo to/from surrounding stations (#7234)
+Add: [Script] Various API functions for managing vehicle groups (#7225, #7336)
+Change: [Win32] Use native GDI engine for rendering fonts (#7572)
+Change: Scale oil refinery edge distance limit by map size (#7514)
+Change: Do not display a news message about old vehicles when a replacement for it is activated (#7401)
+Change: When filtering purchase list by cargo type, buy button now performs a refit if required (#7301)
+Change: Don't apply forbid 90 deg turn settings to ships, and make penalties for turns configurable (#7289, #7372)
+Change: Make the chance of an aeroplane crashing at an airport with a short runway independent of plane crash setting (#7302)
+Change: Non-rectangular catchment area for sparse stations (#7235)
```

Comment:
I kind of want this high on the list as it's an important gameplay change, but it's still just a change.

Reply:
mm, I've gone for "consistently reverse PR number order", rather than making anything special. Could be highlighted in the release announcement, perhaps